### PR TITLE
adjust hp if nil

### PR DESCRIPTION
**Pull request by emilyseibert**
Fixes issue #232 

The HP of wombats is `nil` if the game hasn't started or if the wombat has died. I'm adjusting the HP to either 100 or 0 depending on the scenario.

### CODE REVIEW

File: src/cljs/wombats_web_client/components/game_ranking.cljs [hunk 1]
```diff
@@ -7,25 +7,32 @@
...
+    (let [game-status (:status @info)
+          isStarting? (or (= game-status :pending-open) (= game-status :pending-closed))]
+      [:div {:class-name "game-ranking-box"}
+       [:ul.list-wombat-status
+        (for [{:keys [wombat-name
+                      username
+                      score
+                      hp
+                      color]} @stats]
+          (let [hpNil? (nil? hp)
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/game_ranking.cljs]: hp-nil?

File: src/cljs/wombats_web_client/components/game_ranking.cljs [hunk 2]
```diff
@@ -7,25 +7,32 @@
    (<= 1 hp 29) "dying"
    :else "none"))
 
+(defn get-adjusted-hp [isStarting?]
+  (if isStarting? 100 0))
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/game_ranking.cljs]: along with @oconn's comment, `is-starting?`

File: src/cljs/wombats_web_client/components/game_ranking.cljs [hunk 3]
```diff
@@ -7,25 +7,32 @@
...
-                          [:div.health-bar
-                           [:span.filling {:class (get-health-color hp)
-                                           :style {:width (str hp "%")}}]]
-                          [:div.img-wrapper
-                           [:img {:src (str "/images/wombat_" color "_right.png")}]]
-                          [:div.wombat-name wombat-name]
-                          [:div.username username]
-                          [:div.score score]])]]))
+    (let [game-status (:status @info)
+          isStarting? (or (= game-status :pending-open) (= game-status :pending-closed))]
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/game_ranking.cljs]: `is-starting?`

File: src/cljs/wombats_web_client/components/game_ranking.cljs [hunk 4]
```diff
@@ -7,23 +7,23 @@
...
        [:ul.list-wombat-status
         (for [{:keys [wombat-name
                       username
                       score
                       hp
                       color]} @stats]
-          (let [hpNil? (nil? hp)
-                adjusted-hp-value (if hpNil? (get-adjusted-hp isStarting?) hp)]
+          (let [hp-nil? (nil? hp)
+                adjusted-hp-value (if hp-nil? (get-adjusted-hp is-starting?) hp)]
```
Comment (oconn) [inline src/cljs/wombats_web_client/components/game_ranking.cljs]: so actually on a second pass I have some additional feedback. `is-starting?` is only used inside of `get-adjusted-hp`. All this logic should probably be moved into the body of that function.

**Pull request merged**